astronomer/providers/apache/livy/example_dags/example_livy.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This license shouldn't be here as this project isn't part of ASF and this example is different from OSS too so we don't need that license :)

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/644dc92159393e3dfff0f4eb86da81430e6f6428)

Reply:
This is still unresolved, astronomer/providers/apache/livy/example_dags/example_livy.py still contains the license

Reply:
removed it with [commit](https://github.com/astronomer/astronomer-providers/pull/178/commits/62f2d608dd9c38875fda103dadf36ac60155c91a)